containers/compose.yaml

```diff
@@ -27,7 +27,7 @@ services:
     depends_on:
       postgres:
         condition: service_healthy
-    command: python manage.py runserver 0.0.0.0:${DJANGO_PORT}
+    command: uvicorn mosqlimate.asgi:application --host 0.0.0.0 --port ${DJANGO_PORT}
```

Comment:
That's good, the development server should never be used in production  anyway...